docs/framework_config.rst

```diff
@@ -364,6 +367,28 @@ much the same way they would in V2. The following examples will demonstrate how
         await cog.load_data()
         bot.add_cog(cog)
 
+************************************
+Best practices and performance notes
+************************************
+
+Config prioritizes being a safe data store without developers needing to
+know how end users have configured their bot, or require a database setup.
```

Comment:
The verb tensing in this sentence is wacky, I think just removing the comma and replacing "require" with "requiring" should fix it.